src/Compilers/Test/Core/Compilation/CompilationExtensions.cs

```diff
@@ -371,7 +371,7 @@ void checkControlFlowGraph(IOperation root, ISymbol associatedSymbol)
 
                     case IParameterInitializerOperation parameterInitializerOperation:
                         // https://github.com/dotnet/roslyn/issues/27594 tracks adding support for getting ControlFlowGraph for parameter initializers for local functions.
-                        if ((parameterInitializerOperation.Parameter.ContainingSymbol as IMethodSymbol)?.MethodKind != MethodKind.LocalFunction)
+                        if ((parameterInitializerOperation.Parameter.ContainingSymbol as IMethodSymbol)?.MethodKind is not MethodKind.LocalFunction and not MethodKind.AnonymousFunction)
```

Comment:
```suggestion
                        if ((parameterInitializerOperation.Parameter.ContainingSymbol as IMethodSymbol)?.MethodKind is not (MethodKind.LocalFunction or MethodKind.AnonymousFunction))
```

Reply:
Slight simplification suggestion, feel free to say you prefer it the current way if you want.

Reply:
Yours reads better; I just didn't realize it's a possibility.